xprocspec-runner/pom.xml

```diff
@@ -24,7 +24,7 @@
     <dependency>
       <groupId>org.daisy.xprocspec</groupId>
       <artifactId>xprocspec</artifactId>
-      <version>[1.0.0,2.0.0)</version>
+      <version>1.1.1-SNAPSHOT</version>
```

Comment:
OK. The things you are working on is that also logging related? If not, what about an intermediary release with just the logging stuff?